ui/app/templates/components/scale-events-accordion.hbs

```diff
@@ -0,0 +1,32 @@
+<ListAccordion data-test-scale-events @source={{this.events}} @key="time" as |a|>
+  <a.head @buttonLabel="details" @isExpandable={{a.item.hasMeta}} class="with-columns">
+    <div class="columns inline-definitions">
+      <div class="column is-3">
+        <span class="icon-field">
+          <span class="icon-container" title="{{if a.item.error "Error event"}}" data-test-error={{a.item.error}}>
+            {{#if a.item.error}}{{x-icon "cancel-circle-fill" class="is-danger"}}{{/if}}
+          </span>
+          <span data-test-time title="{{format-ts a.item.time}}">{{format-month-ts a.item.time}}</span>
+        </span>
+      </div>
+      <div class="column is-2">
```

Comment:
Maybe it’s overkill but I hovered over this just to see, a tooltip/title like “Scaled up/down to X” could be nice?

Reply:
I think that could be nice. The autoscaler folks are also looking into improving the messages they emit, so that should help too.